tests/integration/session_management/test_niswitch_multiplexer_reservation .py

```diff
@@ -0,0 +1,137 @@
+import pathlib
+from contextlib import ExitStack
+
+import niswitch
+import pytest
+
+from ni_measurementlink_service.session_management import (
+    PinMapContext,
+    SessionManagementClient,
+)
+from tests.utilities.connection_subset import (
+    ConnectionSubset,
+    get_connection_subset_with_multiplexer,
+)
+from tests.utilities.pin_map_client import PinMapClient
+
+_SITE = 0
+
+
+@pytest.mark.xfail(reason="Requires MeasurementLink 2024Q2 or later.")
```

Comment:
Remind me why we're marking these new tests as failing again? When will we be able to run them on the build machines?